rero_ils/modules/loans/api.py

```diff
@@ -129,6 +132,42 @@ def dumps_for_circulation(self):
             data['pickup_location']['library_name'] = library.get('name')
         return data
 
+    def create_notification(self, notification_type=None):
+        """Creates a recall notification from a checked-out loan."""
+        data = {}
+        if notification_type == 'recall':
```

Comment:
raise an exception if notification_type is not supported

Reply:
coming in the next PR